README.md

```diff
@@ -16,7 +16,13 @@ the square brackets (for multiple-choice questions).
 What are constructor functions and the `new` keyword? What is a prototype? Describe an example of when we would use a constructor function versus a prototype.
 
 Your Answer:
-```text
+```
+A constructor is any function that is used as a constructor. It is used with a 'new' keyword. An example of this would be:
+//var Student = function Student(){
```

Comment:
combing both function expression and function declaration syntax. Typically use function expression notation here like:
```js
var Student = function(){
  //...
}
```